src/python/grpcio_tests/tests/unit/_channel_args_test.py

```diff
@@ -33,6 +33,15 @@ def __int__(self):
     ('arg6', TestPointerWrapper()),
 )
 
+INVALID_TEST_CHANNEL_ARGS = [
+    {
```

Comment:
Nit: please make this test case one-liner like before to be consistent with the rest of the test cases (or will yapf complaining about it?)

Reply:
You are right `yapf` does complaining about it...